modules/lead/product-aws/aws.tf

```diff
@@ -1,31 +1,46 @@
 resource "aws_codebuild_project" "codebuild_build" {
   for_each = var.pipelines
 
-  name          = "${var.product_name}-${each.value.repo}-build"
+  name          = "${each.value.repo}-build"
   description   = "terraform_codebuild_project"
-  build_timeout = "5"
+  build_timeout = "20"
   service_role  = var.codebuild_role
 
   environment {
-    compute_type                = "BUILD_GENERAL1_SMALL"
-    image                       = "aws/codebuild/amazonlinux2-x86_64-standard:2.0"
+    privileged_mode             = true
+    compute_type                = "BUILD_GENERAL1_LARGE"
+    image                       = "489130170427.dkr.ecr.us-east-1.amazonaws.com/builder-image-skaffold:${var.builder_images_version}"
     type                        = "LINUX_CONTAINER"
+    image_pull_credentials_type = "SERVICE_ROLE"
+
+    environment_variable {
+      name  = "SKAFFOLD_DEFAULT_REPO"
+      value = "774051255656.dkr.ecr.us-east-1.amazonaws.com/${var.product_name}"
```

Comment:
won't this be different depending on the environment (sandbox vs prod)?  I think we used to have a variable in lead-environments called `product_image_repo` that we should probably use here to differentiate between environments.

Reply:
changed the value to be passed in from the product_image_repo variable and also passed it in as it's own separate env variable for the buildspec to use